projects/components/src/tabs/content/tab-group.component.ts

```diff
@@ -26,9 +42,38 @@ import { TabComponent } from './tab/tab.component';
     </div>
   `
 })
-export class TabGroupComponent {
+export class TabGroupComponent implements OnChanges, AfterViewInit {
   @ContentChildren(TabComponent)
   public tabs!: QueryList<TabComponent>;
 
+  @Input()
+  public activeLabel?: string;
```

Comment:
this is the label for the activeTab, right? if yes, this should be named `activeTabLabel`

Reply:
Done